plugins/local-storage/tests/local-fs.test.ts

```diff
@@ -182,8 +182,8 @@ describe('Local FS test', () => {
 
       writeTarballStream.on('success', function() {
         const fileLocation: string = path.join(__dirname, '../_storage', newFileName);
+        expect(() => fs.accessSync(fileLocation, fs.constants.R_OK | fs.constants.W_OK)).not.toThrow();
```

Comment:
Please only change `exists` to an alternative or `existsSync`.

Reply:
@DanielRuf Okay, I can make that change. But technically fs.exists also uses fs.access internally.

Reply:
This piece of code already uses `existsSync` so refactoring it is not necessary.

Reply:
@DanielRuf That makes sense, I've made the change and update the PR.